src/themes/greyVest/ListGroupItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import ListItem from './ListItem'
+
+let ListGroupItem = props => (
```

Comment:
having both ListGroupItem and ListItem is really confusing

Reply:
yeah, these are residue from GreyVest's prior state. Just need to figure out if we actually need both, and in what circumstances (eg, if we'll need a new theme key), or whether we can just consolidate them